packages/lib-classifier/src/components/Classifier/components/withKeyZoom/withKeyZoom.js

```diff
@@ -37,13 +37,25 @@ function withKeyZoom (WrappedComponent) {
           return true
         }
         case 'ArrowRight': {
```

Comment:
Arrow navigation mostly works, with the following caveats:

- Up ⬆️ key moves image down ⬇️, and down ⬇️ key moves image up ⬆️... while the ⬅️ left key moves the image ⬅️ left, right ➡️ key moves the image ➡️ right
  -  Basically, there seems to be a mismatch between how the ↕️  vertical and ↔️  horizontal panning is mapped to keys.
- The pan via keys doesn't account for zoom, so a zoomed-out image will move at visually small increments of length, compared to a zoomed-in image. 
  - (I can't tell if zoom makes a difference with panning via mouse drag, though.)
- I'm not sure if the pan accounts for rotation either, so a rotated image might easily 
  - ...but this is something that should only be solved after the PR #1389 (Rotate View) is merged.

Reply:
I've updated the left-right pan so that it matches [Seabird Watch](https://www.zooniverse.org/projects/penguintom79/seabirdwatch/classify). You can test it out on http://localhost:8080/?project=478&env=production and compare with the live project.

Reply:
I'm not sure if we should scale motion, since that would make it hard to move the image by small amounts at high magnifications. I don't think we scale motion in PFE.